Plant-for-the-Planet-org/planet-webapp
# conservation projects in project list

Different marker colour for projects having conservation projects. 
Not sure about the marker colour I chose is desired one. Please help with that.

Changed all projects endpoint to - `/app/projects?filter[purpose]=trees,conservation&_scope=map&currency=${currency}` for conservation projects

**Todo -** 
- [x] redirect donation for conservation projects to planet donations
- [x] Donation URL for Conservation projects is yet to be implemented [Link eg:](https://donate-with-planet-a946yld87-planetapp.vercel.app/?to=conservation-wea-kenya&callback_url=https://planet-webapp-5dh39wlfk-planetapp.vercel.app/&country=DE&currency=EUR&locale=en&tenant=ten_NxJq55pm)
- [ ] ~Project Classification for conservation needs to be implemented~ can be merged without this

### How to Test -

- Open Latest Deployment
- Look for blue color markers on the map
- hover on the marker
- look for unit cost, verify that `amount per m2` is shown instead of `amount per tree`
- click on donate
- verify that unit cost on planet donations is the same as it was on webapp
- complete donation
